### PR TITLE
WEBAPP-524: Update Readme

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

**Comment by klinzo** [comment]
Let me know if anything is missing. Specifically @Taggotty

File: src/modules/common/components/Selector.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ const ActiveElement = styled(Element)`
     : `:hover {
           font-weight: 700;
           border-radius: 0;
-        }`}
+        }`};
```
Comment (Schedulaar) [inline src/modules/common/components/Selector.js:39]:
```suggestion
        }`}
```

File: README.md [hunk 2]
```diff
@@ -2,145 +2,62 @@
...
+[Pull requests](#pull-requests) and [bug reports](#bug-reporting) are also very much appreciated.
+ 
+## Getting Started
+### Prerequisites
+* Install [nodejs](https://nodejs.org/). At least v6 is required, but we recommend the v12 LTS.
+Using the latest version (v13) may lead to errors.
+* Install [yarn](https://yarnpkg.com/)
+
+### IDE
+We suggest [IntelliJ IDEA Ultimate](https://www.jetbrains.com/idea/) as IDE. JetBrains provides free licences for students.
```
Comment (Taggotty) [inline README.md:28]: Maybe you could add plugin suggestions here, if you have any :)

File: docs/02-technology-stack.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+### Testing
+* [Jest](https://facebook.github.io/jest/) is used for testing.
+* [<img src="https://d2ogrdw2mh0rsl.cloudfront.net/production/images/static/header/header-logo.svg" width="150">](https://www.browserstack.com) is used for testing cross-browser compatibility
+* To test another api, you can set the new cms domain via Console of your web browser: Just type `window.localStorage.setItem('api-url', 'https://cms-test.integreat-app.de')` and reload the page.
+
+### Linting
+* The linter for JavaScript is [eslint](http://eslint.org/)
+* The linter for CSS is [stylelint](https://stylelint.io/)
+
+You can run the linter by calling **yarn run lint**. Some issues can be fixed automatically by running **yarn run lintfix**
```
Comment (Taggotty) [inline docs/02-technology-stack.md:31]: did you mean **yarn run lint:fix**

File: README.md (hunk 2 above)
Comment (klinzo) [reply README.md:28]: Plugin suggestions are mentioned below. Apart from those, I personally don't use other plugins (apart from a vim emulation plugin, but I don't consider it a good suggestion here :P)

File: docs/02-technology-stack.md (hunk 3 above)
Comment (klinzo) [reply docs/02-technology-stack.md]: Good catch!

**Pull request merged**